schema-engine/sql-introspection-tests/tests/views/postgresql.rs

```diff
@@ -103,6 +104,80 @@ async fn simple_view_from_one_table(api: &mut TestApi) -> TestResult {
     Ok(())
 }
 
+// PostgreSQL 16 only
```

Comment:
I assume we can not differentiate just the one different expectation depending on the tag? And have to duplicate the whole test?

Reply:
That's what I understand too, last time I asked around, the best was to duplicate the test